src/index.js

```
@@ -1 +1,407 @@
 'use strict';
+
+const express = require('express');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const nodemailer = require('nodemailer');
+const { v4: uuidv4 } = require('uuid');
+
+const app = express();
+const users = [];
+
+// Email configuration
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: process.env.EMAIL,
+    pass: process.env.EMAIL_PASSWORD,
+  },
+});
```

Transporter is created directly from env vars but there's no startup-time validation. Add checks for required env values (EMAIL, EMAIL_PASSWORD, BASE_URL, JWT_SECRET) and fail fast with a clear error if missing so misconfiguration doesn't surface only as sendMail failures at runtime.